stub.py

```diff
@@ -0,0 +1,5 @@
+"""Stub script for pyinstaller."""
+from finesse.__main__ import main
```

Comment:
I think that the options we have are either use absolute imports in `__main__.py` or, indeed, use a stub. Nothing against it.

I would re-organise a bit some things, though, as importing atom `__main__` just does not look correct:

1. Move the function that configures the logger to a `logger.py` file
2. Move the `main` function to `__init__.py`, importing there from logger, main window, etc.
3. Make `__main__.py` and `stub.py` identical, just one doing `from . import main` and the other `from finesse import main`.

Does this make sense?

Reply:
Yup, sounds good!